### DISCUSSION TITLE
Quote character in Inline image title

### DISCUSSION THREAD

**Discussion by Nevinia210** [pnp/pnpcore#992] [Q&A] [answered]
Hello,
i find out that when i try to insert inline image with `"` (`\"` or `&quot;`) character in caption, it results in an error

![image](https://user-images.githubusercontent.com/42325625/195068340-132bc491-8d31-41f5-8543-5a422d5b8290.png)

in UI it is possible to insert quotes in title of inline image.
I think my application internally send json that is not encoded, could you please check it?

**Comment by jansenbe** [comment, marked as answer]
@Nevinia210 : this is a bug, I've created an issue (#995) from this discussion, so let's follow up in that way.
